package.json

```diff
@@ -12,13 +12,15 @@
     "bootstrap": "^5.2.2",
     "date-fns": "^2.29.3",
     "dayjs": "^1.11.5",
+    "formik": "^2.2.9",
     "moment": "^2.29.4",
     "react": "^18.2.0",
     "react-bootstrap": "^2.5.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.4.1",
     "react-scripts": "5.0.1",
-    "web-vitals": "^2.1.4"
+    "web-vitals": "^2.1.4",
```

Comment:
Are we using web-vitals package? If not, we can remove it by running `npm uninstall web-vitals` for a clean package-lock.json 😉 @calovey